civictechprojects/static/css/partials/_LogInController.scss

```
@@ -65,15 +65,43 @@
 }
 
 .LogInController-socialLink {
-  margin: 0 10px;
   background-color: transparent;
   text-decoration: none;
+  border: thin solid;
+  margin-bottom: 10px;
+  width: 325px;
+  margin-left: auto;
+  margin-right: auto;
+
+  a {
+    color: $color-text-dark;
+  }
```

```suggestion
  a {
    color: $color-text-dark;
    padding-left: 15px;
  }
```

Once we remove the text-center class, this will help line up the icons nicely.